leptonic/Cargo.toml

```diff
@@ -30,33 +30,33 @@ js-sys = "0.3.68"
 leptos = "0.6.5"
 leptos_reactive = "0.6.5"
 leptos-tiptap = { version = "0.7.0", optional = true }
-leptos-use = { version = "0.10.2", features = ["math"] }
+leptos-use = { version = "0.12.0", features = ["math"] }
```

Comment:
This is highly unrelated to the `time` issue - Leptonic even build fine with Rust 1.80 and `leptos-use` 0.10.10, though there are warnings.